packages/manager/src/views/AddWebPropertyPage/AddWebProperty.utils.ts

```diff
@@ -9,7 +9,7 @@ export const addNewWebPropertySchema = yup.object({
     .label('Environment Name')
     .noWhitespace()
     .trim()
-    .matches(/^[a-zA-Z0-9-]+$/, 'Only letters, numbers, and dashes are allowed')
+    .matches(/^[a-zA-Z0-9-]+$/, 'Environment is required')
```

Comment:
I wont recomment giving a message like this because 

`.required()` already handles this usecase and says is requied. User's may get confused with this error message as user clearly provided some value